pypeit/par/pypeitpar.py

```diff
@@ -4206,6 +4206,11 @@ def __init__(self, trace_npoly=None, snr_thresh=None, find_trim_edge=None,
         dtypes['find_maxdev'] = [int, float]
         descr['find_maxdev'] = 'Maximum deviation of pixels from polynomial fit to trace used to reject bad pixels in trace fitting.'
 
+        defaults['find_numiterfit'] = 10
```

Comment:
I'm wary of changing the default behavior for all datasets.  Is that justified?  Can we instead set this to 1 (or whatever the equivalent is to recover the current behavior) and then advertise that this option now exists and how it can improve traces?  That will give us some time to decide if we want to change the default behavior.

Reply:
Yes, I think that's a fair point  👍  In principle, a good trace should simply break out of the loop during the 2nd iteration, since the only thing that changes is the initial guess. If the initial guess is close to the final position, nothing _should_ change in the second iteration for a good fit.